usb/esp_tinyusb/include/tinyusb.h

```diff
@@ -34,6 +34,10 @@ typedef struct {
     const uint8_t *configuration_descriptor;   /*!< Pointer to a configuration descriptor. If set to NULL, TinyUSB device will use a default configuration descriptor whose values are set in Kconfig */
     bool self_powered;                         /*!< This is a self-powered USB device. USB VBUS must be monitored. */
     int vbus_monitor_io;                       /*!< GPIO for VBUS monitoring. Ignored if not self_powered. */
+    bool no_backround_task;                    /*!< When set to false, background task handling tinyusb events is created. */
+    uint32_t task_priority;                    /*!< Task priority of created background tinyusb task */
+    UBaseType_t stack_size;                    /*!< Stack size of crated background tinyusb task */
+    BaseType_t core_id;                        /*!< Select core on which background tinyusb task will run or tskNO_AFFINITY  */
```

Comment:
In most of our components we use type `size_t`. This way, we don't have to include freertos headers in this file

Reply:
I have changed the type of `task_priority` and `stack_size` to `size_t`
For core_id, I have kept it `BaseType_t `